verified-reviews/mod.ts

```diff
@@ -0,0 +1,18 @@
+import type { App, AppContext as AC } from "deco/mod.ts";
+import manifest, { Manifest } from "./manifest.gen.ts";
+export interface ConfigVerifiedReviews {
+	idWebsite: string;
+	secretKey?: string;
+	plateforme?: string;
+}
+
+/**
+ * @title App Opiniões Verificadas
+ */
+export default function App(
+	state: ConfigVerifiedReviews
+): App<Manifest, ConfigVerifiedReviews> {
+	return { manifest, state };
```

Comment:
I think you're using `8` spaces for indentation ? could you check it please? run `deno task check/fmt` in your created files